opencog/nlp/types/atom_types.script

```diff
@@ -118,6 +118,7 @@ LG_CONN_DIR_NODE <- NODE    // e.g. "+"
 // Consists of the connector node, the direction, and possibly the
 // multi-connector node.
 LG_CONNECTOR <- LINK   // e.g. "MX+"
+MST_Connector <- LINK "MSTConnector"
 
```

Comment:
Rohit,

Good practice for a project like this is to always merge the changes from opencog/master and then run:

```
make test
```

to make sure that you didn't break any of the OpenCog unit tests before submitting a pull request. If you go to:

http://buildbot.opencog.org:8010/waterfall

you can see the status of the builds off of new commits to opencog/master run by buildbot. If one or more of the tests are failing you'll see the red indicator for the build and you can look at the `stdio` output to see what test failed. It will also fail in `make all` if someone breaks the build itself.

So if `make test` fails on your machine, you can check to see if the test that is failing is the one that already fails on the buildbot. If so, then your changes probably didn't cause the failure, especially if the test is for an area of OpenCog where you are not developing. Once you have established that you didn't break the build and you didn't break any unit tests, then you submit the pull request.

Additionally, I always like to check the buildbot after my pull request has been merged to make sure that it also passes the tests and the build works, just in case. In the event that I do break something, I prefer to have the fix ready before anyone else notices. ;-)